specifyweb/stored_queries/query_construct.py

```diff
@@ -71,25 +71,8 @@ def handle_tree_field(self, node, table, tree_rank, next_join_path, current_fiel
 
         assert len(treedefs_with_ranks) >= 1, "Didn't find the tree rank across any tree"
 
-        def make_tree_field_spec(tree_node):
-            return current_field_spec._replace(
-                root_table=table, # rebasing the query
-                root_sql_table=tree_node, # this is needed to preserve SQL aliased going to next part
-                join_path=next_join_path, # slicing join path to begin from after the tree
-            )
-
-        # TODO: Verify below code can be removed
-        # cases = []
-        # field = None # just to stop mypy from complaining.
-        # for ancestor in ancestors:
-        #     field_spec = make_tree_field_spec(ancestor)
-        #     query, orm_field, field, table = field_spec.add_spec_to_query(query)
-        #     #field and table won't matter. rank acts as fork, and these two will be same across siblings
-        #     cases.append((getattr(ancestor, treedefitem_column) == treedefitem_param, orm_field))
-
-        # column = sql.case(cases)
-
-        # return query, column, field, table
+        column_name = next_join_path[0].name
```

Comment:
Hmm could you leave a comment to revert back to the previous implementation in 4929? You'd need that to support multiple trees in batch-edit. I understand if you don't want to add that, but at least keep the commented out the code for future reference.